src/building/dock.c

```diff
@@ -9,6 +9,11 @@
 #include "map/routing.h"
 #include "map/terrain.h"
 #include "scenario/map.h"
+#include "empire/empire.h"
+#include "figure/trader.h"
+#include "figuretype/trader.h"
+
+#define MAX_DISTANCE_FOR_REROUTING_ANCHORED 50
 
 int building_dock_count_idle_dockers(const building *dock)
```

Comment:
@crudelios moved static functions definition before it's called, and removed progressive declaration.